script.js

```diff
@@ -0,0 +1,17 @@
+const burger = document.getElementById('burger');
+const menuItems = document.getElementById('menu-block');
+const close = document.getElementById('close');
+const item1 = document.getElementById('menuitem1');
+const item2 = document.getElementById('menuitem2');
+const item3 = document.getElementById('menuitem3');
+function show() {
+  menuItems.style.display = 'flex';
+}
+function hide() {
+  menuItems.style.display = 'none';
+}
+burger.addEventListener('click', show);
+close.addEventListener('click', hide);
+item1.addEventListener('click', hide);
+item2.addEventListener('click', hide);
+item3.addEventListener('click', hide);
```

Comment:
- [ ] Kindly try to match the Figma design in `color`, `font-size`, `font-weight` and decrease the spacing between menu links.
- [ ] Please make the first letter in the menu links a capital letter.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/167650875-f949361d-409a-4c37-9860-596d1bd213d1.png) | ![image](https://user-images.githubusercontent.com/24703171/167650997-78f72c60-e828-49dc-88f2-7649dc43d9c1.png) |